.github/CODEOWNERS

```diff
@@ -134,8 +134,6 @@ Lib/ast.py                    @isidentical
 
 **/*typing*                   @gvanrossum @Fidget-Spinner @JelleZijlstra
 
-**/*asyncore                  @giampaolo
```

Comment:
Not sure if this should be done or not.

Reply:
I suggest keeping them in (unless @giampaolo prefers removing).

The code is still used (albeit minimally), and until it is fully removed, it would be good to keep it maintained to some degree.